### DISCUSSION TITLE
where is file of weights

### DISCUSSION THREAD

**Discussion by DetectiveLin** [open-edge-platform/anomalib#1086] [Q&A]
after i trained data, where is the file of weights ??

**Comment by yonafalinie** [comment]
Hai, after you trained the data, the weights file will be save under 'results' folder with  corresponding anomaly model folder

**Comment by samet-akcay** [comment]
As @yonafalinie, pointed out, the weights will be saved into `results` directory. If you would like to change the path, you could do it here.
https://github.com/openvinotoolkit/anomalib/blob/f0b2db8565a20a4d23f29758448740324f9378fc/src/anomalib/models/patchcore/config.yaml#L60
